engine/class_modules/sc_death_knight.cpp

```diff
@@ -5117,6 +5130,7 @@ struct howling_blast_t : public death_knight_spell_t
     if ( p() -> buffs.rime -> up() )
     {
       m *= 1.0 + p() -> buffs.rime -> data().effectN( 2 ).percent();
+      m *= 1.0 + p() -> spec.rime_rank_2 -> effectN( 1 ).percent();
```

Comment:
Had to manually implement the rank as `apply_affecting_aura` didn't seem to buff Howling Blast like it should.  My guess at this point was that `apply_affecting_aura` is meant for direct spells and that I should manually implement rank 2s that affect buffs.

Reply:
Rime's ranks 1 and 2 are additive, you should put both modifiers on the same line to avoid multiplying them.